draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/MobileActions.tsx

```diff
@@ -79,7 +68,7 @@ const renderDefaultLinkOrAction = (
   if (kind === "link" && buttonIsLink(defaultAction) && defaultAction.href) {
     return (
       <MenuItem
-        action={defaultAction.href}
+        href={defaultAction.href}
```

Comment:
Can MenuItems have `onClicks` _and_ `hrefs`? Like, it's a link style menu item but also I wanna do some analytics stuff onClick or maybe load a modal under certain conditions.

Or actually, even in the case of performance, if there's a basepath-ified href we pass, but the onClick we pass has a different URL cos we're using the basepath aware history. 😖

Reply:
Ah yeah nvm ignore me, it's covered below.